lib/ansible/modules/cloud/docker/docker_secret.py

```diff
@@ -72,10 +84,14 @@
 
 EXAMPLES = '''
 
-- name: Create secret foo
-  docker_secret:
+- name: Create secret foo (from a file on the control machine)
+  docker_config:
```

Comment:
This seems like an accidental copy/paste

Reply:
Indeed! Thanks for spotting this!